tools/ci/cirrus_wheels.yml

```
@@ -85,13 +85,17 @@ macosx_arm64_task:
     
     ver=$(sw_vers -productVersion)
     macos_target=$(python -c "print('14.0') if '$ver' >= '14.0' else print('11.0')")
-    if [ macos_target == '14.0' ]; then
+    if [ $macos_target == '14.0' ]; then
         export INSTALL_OPENBLAS=false
+        export INSTALL_GFORTRAN=false
     else
         export INSTALL_OPENBLAS=true
+        # scipy-openblas wheels seem to need this compiler rather than
```

Once we move to using the scipy-openblas wheels, we can remove this.

```suggestion
        # scipy-openblas tarballs need the gfortran shared object
        # from here rather than
```